### PR TITLE
feat: add additional build files as examples

### PR DESCRIPTION
**Pull request by Phylu**
I have added further build files based on the integration sections on the preferences page of the Crashtest Security Suite.

### CODE REVIEW

File: .github/main.yml [hunk 1]
```diff
@@ -0,0 +1,18 @@
+on: [push]
+
+jobs:
+  hello_world_job:
```
Comment (Dalabad) [inline .github/main.yml]: i guess you missed renaming this?

File: .github/main.yml [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+    runs-on: ubuntu-latest
+    name: Run Crashtest Security Scan
+    steps:
+    - name: Crashtest Security Action Step
+      id: crashtest-security
+      uses: crashtest-security/crashtest-security-action@v1.0.0
+      with:
+        crashtest-webhook: '${{ secrets.CRASHTEST_WEBHOOK }}'
+        pull-report: 'true'
```
Comment (Dalabad) [inline .github/main.yml:13]: does this need to be a string?

File: azure-pipelines.yml [hunk 3]
```diff
@@ -0,0 +1,12 @@
+jobs:
+- job: CashtestSecurityScan
```
Comment (Dalabad) [inline azure-pipelines.yml:2]: can this not include any spaces?

File: cloudbuild.yaml [hunk 4]
```diff
@@ -0,0 +1,7 @@
+steps:
```
Comment (Dalabad) [inline cloudbuild.yaml:1]: why is this file named `.yaml` and not `.yml` like the others?

File: start_crashtest_codeship.sh [hunk 5]
```diff
@@ -0,0 +1,54 @@
+#!/usr/bin/env bash
```
Comment (Dalabad) [inline start_crashtest_codeship.sh:1]: is there no `.yml` file necessary for codeship?

**Review by Dalabad — COMMENTED** [review]
Github recommends to put a linebreak in all files at the last line

**Comment by Phylu** [comment]
> Github recommends to put a linebreak in all files at the last line

Nice. I will fix this. I am just wondering: Where do I see the recommendation?

File: .github/main.yml (hunk 1 above)
Comment (Phylu) [reply .github/main.yml]: I will update this.

File: .github/main.yml (hunk 2 above)
Comment (Phylu) [reply .github/main.yml:13]: Yes. I tried it as a boolean first and it did not work.

File: azure-pipelines.yml (hunk 3 above)
Comment (Phylu) [reply azure-pipelines.yml:2]: This is the naming scheme proposed by the documentation.

File: cloudbuild.yaml (hunk 4 above)
Comment (Phylu) [reply cloudbuild.yaml:1]: I followed the official namings here and this file is supposed to have the `.yaml` suffix.

File: start_crashtest_codeship.sh (hunk 5 above)
Comment (Phylu) [reply start_crashtest_codeship.sh:1]: Unfortunately, for Codeship Basic, there is no YAML file possible to be configured. It needs to be done via their Frontend.

**Review by Dalabad — APPROVED** [review]

**Pull request merged**